javascript/javascript1/week4/class_exercises/sentiment_analyzer.js

```diff
@@ -0,0 +1,37 @@
+const positivePossibilities = [
+    "mega", "awesome", "super", "excited", "positive", "optimistic", "good", "excellent", "energetic",
+];
+
+const negativePossibilities = [
+    "angry", "negative", "pessimistic", "sad", "frustrated", "bad", "down", "sick", "tired",
+];
+
+
+function wordMatcher (wordsPossibilities, sentence) {
+    const matchesArray = [];
+    for (word of wordsPossibilities) {
+        if (sentence.includes(word)) {
+            matchesArray.push(word);
+        }
+    }
+    return matchesArray;
+    }
+
+
+function getSentimentScore(sentence, wordsPossibilities1, wordsPossibilities2) {
+    
+    const positiveWords = wordMatcher(wordsPossibilities1, sentence);
+    const negativeWords = wordMatcher(wordsPossibilities2, sentence);
+
+    return {
+        score: positiveWords.length - negativeWords.length,
+        positiveWords,
+        negativeWords,
+    }    
+}
+
+const sentimentScoreObject1 = getSentimentScore(
+    'I am mega super awesome happy, although yesterday was a bad day because I felt sick and tired',
```

Comment:
What would happen if I wrote SUPER in uppercase or started my sentence with Mega? 😵‍💫

Converting your string to lowercase or uppercase can prevent some words from slipping through during the comparison. :)